scipy/optimize/tests/test__numdiff.py

```diff
@@ -267,16 +270,52 @@ def test_scalar_scalar_abs_step(self):
 
     def test_scalar_vector(self):
         x0 = 0.5
-        jac_diff_2 = approx_derivative(self.fun_scalar_vector, x0,
-                                       method='2-point')
-        jac_diff_3 = approx_derivative(self.fun_scalar_vector, x0)
+        with MapWrapper(2) as mapper:
+            jac_diff_2 = approx_derivative(self.fun_scalar_vector, x0,
+                                           method='2-point', workers=mapper)
+        jac_diff_3 = approx_derivative(self.fun_scalar_vector, x0, workers=map)
         jac_diff_4 = approx_derivative(self.fun_scalar_vector, x0,
-                                       method='cs')
+                                       method='cs', workers=None)
         jac_true = self.jac_scalar_vector(np.atleast_1d(x0))
         assert_allclose(jac_diff_2, jac_true, rtol=1e-6)
         assert_allclose(jac_diff_3, jac_true, rtol=1e-9)
         assert_allclose(jac_diff_4, jac_true, rtol=1e-12)
 
+    @pytest.mark.fail_slow(5.0)
+    def test_workers(self):
+        # check that nfev consumed by approx_derivative is tracked properly
```

Comment:
Purpose of test is to check that series and parallel evaluation of derivatives gives the same number of function evaluations, and the same derivative.